libs/types/src/system_settings.ts

```diff
@@ -64,6 +64,16 @@ export interface SystemSettings {
   readonly precinctScanAdjudicationReasons: readonly AdjudicationReason[];
   readonly precinctScanDisallowCastingOvervotes: boolean;
   readonly precinctScanEnableShoeshineMode?: boolean;
+  /**
+   * Excludes original snapshots in cast vote record reports, decreasing export size and
+   * import/export time
+   */
+  readonly castVoteRecordOptimizationExcludeOriginalSnapshots?: boolean;
+  /**
+   * Excludes redundant metadata in cast vote record reports, decreasing export size and
+   * import/export time
+   */
+  readonly castVoteRecordOptimizationExcludeRedundantMetadata?: boolean;
```

Comment:
Can we make it such that, if you don't include these fields in the `systemSettings.json`, we default to the behavior that we want in the overwhelming majority of cases (i.e. excluding this data)? (We really only want to include this data during cert and, even then, for just a tiny part of cert.)

Could do that by flipping these booleans, e.g., `castVoteRecordOptimizationIncludeOriginalSnapshots` and `castVoteRecordOptimizationIncludeRedundantMetadata`

Reply:
I guess `Optimization` doesn't really make sense in the names anymore if you flip haha

Reply:
Would it be worth also just collapsing them into one flag, like `castVoteRecordStrictCdf` or something?

Reply:
Yeah I like that!

Reply:
Actually, on second thought, let's keep them separate. Just to have the flexibility. There may be cases where a customer wants original snapshots (for auditing purposes) but not the redundant metadata.

It's also not clear to me reading the CVR CDF spec whether or not original snapshots are required or optional.